tenable/sc/organization.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+"""
+"""
+from tenable.sc.base import SCEndpoint
+
+
+
+class OrganizationAPI(SCEndpoint):
+
+    def _constructor(self, **kw):
+        '''
+        Organization document constructor
+        '''
+        if 'name' in kw:
+            kw["name"] = self._check('name', kw['name'], str)
```

Comment:
If simply validating that the name param is a string, no need to overload it with itself.  `self._check('name', kw['name'], str)` is enough.  Also please comment on the action, in this case something like `# Verifying the name attribute to be a string`

Reply:
done